test/src/unit-capi-consolidation.cc

```diff
@@ -7587,29 +7587,6 @@ TEST_CASE_METHOD(
   }
 }
 
-TEST_CASE_METHOD(
```

Comment:
This definitely looks like a test we should have, and in fact we should probably have more like this with large cell sizes.  What if a single cell has a 10+MB value?  1G+?

We should be able to make progress as long as a cell fits in the memory budget.